src/components/PaginationUI.js

```diff
@@ -0,0 +1,43 @@
+import renderGames from '../modules/RenderGames.js';
+import gamesApi from '../modules/GamesAPI.js';
+
+let { pageNumber } = gamesApi;
+
+const renderPagination = () => {
+  const mainSection = document.querySelector('#main');
+  const paginationContainer = document.createElement('div');
+  const pageNumberLabel = document.createElement('p');
+  const btnPrevious = document.createElement('button');
+  const btnNext = document.createElement('button');
+
+  btnPrevious.classList.add('btn', 'btn-previous');
+  btnPrevious.textContent = '<< Prev';
+  pageNumberLabel.textContent = ` - ${pageNumber >= 0 ? pageNumber + 1 : 1} -`;
+  btnNext.classList.add('btn', 'btn-next');
+  btnNext.textContent = 'Next >>';
+  paginationContainer.classList.add('pagination-container');
+  paginationContainer.appendChild(btnPrevious);
+  paginationContainer.appendChild(pageNumberLabel);
+  paginationContainer.appendChild(btnNext);
+
+  mainSection.appendChild(paginationContainer);
+
+  btnNext.addEventListener('click', () => {
+    const gameListContainer = document.querySelector('.games-list');
+    gameListContainer.innerHTML = '';
+    pageNumber = (pageNumber + 1) < (gamesApi.getNumberOfGames() / 15)
+      ? pageNumber + 1 : gamesApi.getNumberOfGames / 15;
+    pageNumberLabel.textContent = ` - ${pageNumber + 1} -`;
+    renderGames(pageNumber);
+  });
```

Comment:
There seems to be one small problem on the PaginationUI. When I run it it gives a Nan object displayed and when I console logged the pageNumber value it gives undefined. 

If I press "prev" and press "next" it works again. It seems the event listener is picking the wrong figures on your ternary statement on the first page run. Try it again......could be my local server with the problem. 

See screenshot: 
- Gave a pagenumber of zero on initial load 
- when I pressed "next" gave an undefined value 

![image](https://user-images.githubusercontent.com/44676055/152348813-65d450c7-caa5-41c0-bc4d-4f8a30d192f8.png)

Reply:
Thank you, Alick. That was a good catch. 
I saw it a couple of times, but I didn't fix it because I was not able to regenerate the error. I am working on it now.

Reply:
Hi @Beardless-sheik 
The error is now fixed. Besides, functionality to disable `prev` button on first page and `next` button on last page is added.